tests/integration/dynamic_ip_test.py

```diff
@@ -70,7 +70,10 @@
 IPV6_DEFAULT_GATEWAY = "::/0"
 IPV4_DEFAULT_GATEWAY = "0.0.0.0/0"
 
+LEASE_FILE = "/tmp/nmstate.dnsmasq.lease"
+
 DNSMASQ_CONF_STR = """
```

Comment:
You can probably make this a format string and use the constants directly.

Reply:
The line length  will exceeded the 78 which looks ugly in my vim screen.

Reply:
ok